src/sys/socket/consts.rs

```diff
@@ -109,6 +109,7 @@ mod os {
 
     // Ancillary message types
     pub const SCM_RIGHTS: c_int = libc::SCM_RIGHTS;
+    pub const SCM_TIMESTAMP: c_int = SO_TIMESTAMP;
```

Comment:
This constant is declared in `libc`, so it should reference that constant instead of defining it directly.